TicTacToeApp/Board.swift

```diff
@@ -0,0 +1,48 @@
+
+public class Board {
```

Comment:
Righto. I'm going to bring up leveraging named parameters more with you in person, since I'm not quite sure what that would entail.